niworkflows/interfaces/freesurfer.py

```diff
@@ -294,7 +294,7 @@ def _run_interface(self, runtime):
 class _MedialNaNsInputSpec(BaseInterfaceInputSpec):
     in_file = File(exists=True, mandatory=True, desc='input surface file')
     subjects_dir = Directory(mandatory=True, desc='FreeSurfer SUBJECTS_DIR')
-    density = traits.Enum('32', '59', '164', desc="Input file density (fsLR only)")
+    density = traits.Enum(32, 59, 164, desc="Input file density (fsLR only)")
```

Comment:
```suggestion
    density = traits.Enum('32k', '59k', '164k', desc="Input file density (fsLR only)")
```
density is string (actually, the resolution should be string too).

Reply:
Yup, I think as string would be best - I would love to just pass in the `den-` key but
```
In [1]: import templateflow.api as tf                                                                                                                         

In [2]: tf.get('fsLR', density='32k')                                                                                                                         
Out[2]: []

In [3]: tf.get('fsLR', density=32)  # or '32'
Downloading https://templateflow.s3.amazonaws.com/tpl-fsLR/tpl-fsLR_hemi-L_den-32k_nomedialwall.surf.gii
...
```

Reply:
ugh, can you file an issue in templateflow/python-client?